tests/cross/src/test/scala/tests/pc/CompletionKeywordSuite.scala

```diff
@@ -485,12 +485,29 @@ class CompletionKeywordSuite extends BaseCompletionSuite {
       |""".stripMargin,
   )
 
+  check(
+    "extends",
+    """
+      |package foo
+      |
+      |class Foo ext@@{
+      |}
+    """.stripMargin,
+    """|extension
+       |extends
+       |""".stripMargin,
+    compat = Map(
+      "2" -> "extends"
+    ),
+  )
+
   check(
     "topLevel",
     "@@",
     """|abstract class
        |case class
        |class
+       |extends
```

Comment:
That's the biggest issue here actually. We don't want to show extends if it's not after an identifier. `extend` cannot live alone.

Reply:
But the `path` is `List()` :(( 
How can this be checked?

Reply:
I can try to add a flag like `isAfterIdentifier`?

Reply:
I think one suggestion was to use diagnostics and see if there is an error, which has the current position and we can use.

Reply:
The other one as mentioned by you in the metals standup was using tokenisation to see where we are in the file.